### PR TITLE
Add Flink accumulators

### PR DESCRIPTION
**Pull request by gesundkrank**
This PR adds accumulators to Flink BeamSink. They will provide some more insights to the user.

It also updates the readme.

### CODE REVIEW

File: flink/src/main/scala/com/metamx/tranquility/flink/BeamSink.scala [hunk 1]
```diff
@@ -34,16 +38,24 @@ class BeamSink[T](beamFactory: BeamFactory[T])
 {
   var sender: Option[Tranquilizer[T]] = None
 
-  private val exception = new AtomicReference[Throwable]()
+  private val exception       = new AtomicReference[Throwable]()
+  private val sentCounter     = new LongCounter()
```
Comment (gianm) [inline flink/src/main/scala/com/metamx/tranquility/flink/BeamSink.scala]: In tranquility-kafka and tranquility-server, what you call "sent" and "accepted" are called "received" and "sent". Could you change these to be consistent?

**Comment by gianm** [comment]
looks good after adjusting the naming of the counters.

**Comment by gesundkrank** [comment]
Fixed your comment. Thank you for the review!

**Comment by gianm** [comment]
:+1: Looks good to me!

**Comment by fjy** [comment]
:+1:

**Pull request merged**